### PR TITLE
Add Tamriel Rebuilt

### PR DESCRIPTION
**Pull request by IllusiveMan196**
Add mods
Add location
Add loadorder (report if anything breaks)

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -881,6 +881,35 @@ plugins:
...
+    after:
+      - 'Beautiful cities of Morrowind.ESP'
+      - 'TR_Travels.esp'
+  - name: 'TR_Travels Patch.esp'
+    url: [ 'https://www.nexusmods.com/morrowind/mods/42145/' ]
+    after:
+      - 'Beautiful cities of Morrowind.ESP'
+      - 'TR_Travels.esp'
+      - 'TR_Travels_(Preview_and_Mainland).esp'
+  - name: 'TR_Travels_(P_M) Patch.esp'
```
Comment (MacSplody) [inline masterlist.yaml]: I can't find that file `TR_Travels_(P_M) Patch.esp`, where is it hosted?

Comment (IllusiveMan196) [reply masterlist.yaml]: Turns out it is BCOM's patch for TR_Travels

Comment (IllusiveMan196) [reply masterlist.yaml]: So that would be here https://www.nexusmods.com/morrowind/mods/49231

Comment (MacSplody) [reply masterlist.yaml]: Right, in that case can you move that patch to the BCOM files and update the link.

Comment (IllusiveMan196) [reply masterlist.yaml]: Done and done

**Review by MacSplody — APPROVED** [review]

**Pull request merged**